packages/shr-fhir-export/lib/extensions.js

```diff
@@ -538,6 +552,54 @@ class ExtensionExporter {
     }
   }
 
+  inlineSubextension(subExt, extension, field, baseId, baseExt) {
+    for (const ext of subExt.snapshot.element) {
+      const snapExt = common.cloneJSON(ext);
+      // Slice off the extraneous "Extension" that will start the id
+      const splitPoint = snapExt.id.indexOf('.');
+      const endId = snapExt.id.slice(splitPoint + 1);
+      // Construct a newId and path for the inlined version of the extension
+      const newId = `${baseId}.extension:${common.shortID(field.effectiveIdentifier)}.${endId}`;
```

Comment:
The part after the `:` should match the slice name.  Since you changed the slice names to use camelCase, then you need to use the camelCase short ID after the `:` too.

Reply:
And I guess if there is any code anywhere else that is relying on the form of that `id` (I'm not sure if there is though), we'd want to make sure that's expecting camelCase slice names too.  Might not be a problem -- I don't remember if we ever use the ids in lookups.

Reply:
Makes sense, I made the update so that IDs had camelCase.